src/main/java/org/springframework/hateoas/MediaTypes.java

```diff
@@ -34,4 +35,14 @@
 	 * Public constant media type for {@code application/hal+json}.
 	 */
 	public static final MediaType HAL_JSON = MediaType.valueOf(HAL_JSON_VALUE);
+	
+	/**
+	 * A String equivalent of {@link MediaTypes#HAL_JSON_UTF8}.
+	 */
+	public static final String HAL_JSON_UTF8_VALUE = HAL_JSON_VALUE + ";charset=UTF-8";
```

Comment:
Thanks! Yeah, I had to do the same to have a `hal+json` media type with `;charset=UTF-8`. Hopefully this gets in the next release.

I agree it's an odd way to write it but in this situation I'd rather keep it as Spring's `MediaType` does rather than be different only here.